### PR TITLE
Add Windows 10 Screenshots

### PR DESCRIPTION
**Pull request by Daltz333**
Closes #695

### CODE REVIEW

File: source/docs/networking/networking-introduction/windows-firewall-configuration.rst [hunk 1]
```diff
@@ -6,51 +6,95 @@ Many of the programming tools used in FRC need network access for various reason
...
 
-.. image:: images/windows-firewall-configuration-1.png
+      .. image:: images/windows-firewall-configuration-6.png
 
-Click ``Start`` -> ``Control Panel`` to open the Control Panel. Click the dropdown next to ``View by:`` and select ``Small icons`` then click ``Windows Defender Firewall``.
+   .. group-tab:: Windows 7
 
-Allow a program...
-^^^^^^^^^^^^^^^^^^
+      Click :guilabel:`Start` -> :guilabel`Control Panel` to open the Control Panel. Click the dropdown next to :guilabel:`View by:` and select :guilabel:`Small icons` then click :guilabel:`Windows Defender Firewall`.
```
Comment (sciencewhiz) [inline source/docs/networking/networking-introduction/windows-firewall-configuration.rst:90]:
```suggestion
      Click :guilabel:`Start` -> :guilabel:`Control Panel` to open the Control Panel. Click the dropdown next to :guilabel:`View by:` and select :guilabel:`Small icons` then click :guilabel:`Windows Defender Firewall`.
```

File: source/docs/networking/networking-introduction/windows-firewall-configuration.rst [hunk 2]
```diff
@@ -6,51 +6,95 @@ Many of the programming tools used in FRC need network access for various reason
 Disabling Windows Firewall
 --------------------------
 
+.. important:: Disabling your firewall requires administrator privelages to the PC. Additionally note that disabling the firewall is not recommended for computers that often connect to the internet.
```
Comment (sciencewhiz) [inline source/docs/networking/networking-introduction/windows-firewall-configuration.rst:9]:
```suggestion
.. important:: Disabling your firewall requires administrator privileges to the PC. Additionally note that disabling the firewall is not recommended for computers that connect to the internet.
```

File: source/docs/networking/networking-introduction/windows-firewall-configuration.rst [hunk 3]
```diff
@@ -6,51 +6,95 @@ Many of the programming tools used in FRC need network access for various reason
...
 
-Alternatively, you can add exceptions to the Firewall for any FRC programs you are having issues with.
+      .. image:: images/windows-firewall/add-to-firewall.png
 
-Open Control Panel
-^^^^^^^^^^^^^^^^^^
+      For each FRC program you are having an issue with, make sure that it appears in the list and that it has a check in each of the 3 columns. If you need to change a setting, you made need to click the :guilabel:`Change settings` button in the top right before changing the settings. If the program is not in the list at all, click the :guilabel:`Allow another program...` button and browse to the location of the program to add it.
 
-.. image:: images/windows-firewall-configuration-1.png
+      .. image:: images/windows-firewall-configuration-6.png
```
Comment (sciencewhiz) [inline source/docs/networking/networking-introduction/windows-firewall-configuration.rst:86]: Windows 10 screenshot?

Comment (Daltz333) [reply source/docs/networking/networking-introduction/windows-firewall-configuration.rst:86]: It's the same look on Windows 10

Comment (sciencewhiz) [reply source/docs/networking/networking-introduction/windows-firewall-configuration.rst:86]: Mine looks slightly different. However, even if the content is the same, the the theme change is jarring

File: source/docs/networking/networking-introduction/windows-firewall-configuration.rst [hunk 4]
```diff
@@ -1,56 +1,100 @@
...
 
-.. image:: images/windows-firewall-configuration-2.png
+      Click :guilabel:`Start` -> :guilabel:`Control Panel` to open the Control Panel. Click the dropdown next to :guilabel:`View by:` and select :guilabel:`Small icons` then click :guilabel:`Windows Defender Firewall`.
 
-In the left pane, click ``Turn Windows Defender Firewall on or off``, and click yes. Enter your Administrator password if a dialog appears.
+      .. image:: images/windows-firewall-configuration-2.png
 
-Disable the Firewall
-^^^^^^^^^^^^^^^^^^^^
+      In the left pane, click :guilabel`Turn Windows Defender Firewall on or off`, and click yes. Enter your Administrator password if a dialog appears.
```
Comment (sciencewhiz) [inline source/docs/networking/networking-introduction/windows-firewall-configuration.rst:49]:
```suggestion
      In the left pane, click :guilabel:`Turn Windows Defender Firewall on or off`, and click yes. Enter your Administrator password if a dialog appears.
```

**Pull request merged**